### PR TITLE
retrained bert_tiny_uncased_en_sst2_training.ipynb

### PR DESCRIPTION
**Pull request by susnato**
### What does this PR do?
As discussed in this [issue](https://github.com/keras-team/keras-nlp/issues/542), this PR updates the previous [training script](https://github.com/keras-team/keras-nlp/blob/master/tools/checkpoint_training/bert_tiny_uncased_en_sst2_training.ipynb) for finetuning BERT on SST2. 

The colab link is this : https://colab.research.google.com/drive/1afTO0ahF3vZrJtkVSGXwBLV2OGeDfomL?usp=sharing

Valdiation Scores achieved : 
```
Epoch 1/2
4210/4210 [==============================] - 828s 191ms/step - loss: 0.3782 - sparse_categorical_accuracy: 0.8299 - val_loss: 0.4344 - val_sparse_categorical_accuracy: 0.8165
Epoch 2/2
4210/4210 [==============================] - 783s 186ms/step - loss: 0.2409 - sparse_categorical_accuracy: 0.9039 - val_loss: 0.4626 - val_sparse_categorical_accuracy: 0.8222
```

@chenmoneygithub

### CODE REVIEW

**Comment by susnato** [comment]
I signed the form so `cla/google` and it's green now.

**Comment by susnato** [comment]
Hi @chenmoneygithub I finetuned `BertClassifier` again on `SST2` as you said. Please check it.

**Comment by chenmoneygithub** [comment]
@susnato Thanks a lot! Looks beautiful overall!

Another thing you may want to do is to use a decayed learning rate:
```
lr = tf.keras.optimizers.schedules.PolynomialDecay(
    5e-5,
    decay_steps={total_training_steps},
    end_learning_rate=0.0,
)
```

Please let me know how it works, thx again!

**Comment by mattdangerw** [comment]
Just a drive by comment, but we should replace, not copy, the old colab with this PR. We don't want to keep spawning versions of the colab each time we update something.

**Comment by chenmoneygithub** [comment]
@susnato Hi! sorry for the late reply, I was on vacation the last week. This is a bit odd to me, in my experiments the decayed lr generally works better (but I tried it with BERT base model). If this is stable on your side, it sounds good to stay with the constant `lr`.

File: tools/checkpoint_training/bert_tiny_uncased_en_sst2_training.ipynb [hunk 1]
```diff
@@ -1,4061 +1,401 @@
...
+      "outputs": []
     }
-   }
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 0
-}
+  ]
+}
```
Comment (chenmoneygithub) [inline tools/checkpoint_training/bert_tiny_uncased_en_sst2_training.ipynb:401]: nit: newline at the end

File: tools/checkpoint_training/bert_tiny_uncased_en_sst2_training.ipynb [hunk 2]
```diff
@@ -1,4061 +1,401 @@
...
-      "overflow": null,
-      "overflow_x": null,
-      "overflow_y": null,
-      "padding": null,
-      "right": null,
-      "top": null,
-      "visibility": null,
-      "width": "20px"
-     }
+      "cell_type": "code",
```
Comment (chenmoneygithub) [inline tools/checkpoint_training/bert_tiny_uncased_en_sst2_training.ipynb:392]: nit: delete the empty cell

**Review by chenmoneygithub — CHANGES_REQUESTED** [review]

**Comment by susnato** [comment]
Hi @chenmoneygithub thanks a lot for your comments! I made the changes you requested, please check them.

**Review by chenmoneygithub — APPROVED** [review]

**Comment by chenmoneygithub** [comment]
@susnato Oh one more thing - could you run ./shell/format.sh in your branch? the style check is broken.

**Comment by susnato** [comment]
@chenmoneygithub Done!

**Comment by susnato** [comment]
Hi @chenmoneygithub the `Check the code format` is passed! but there seems to be an error with `
keras-nlp-accelerator-testing` is this something related to my code?

**Comment by chenmoneygithub** [comment]
@susnato That one is okay, don't worry.
